src/app/core/signals/remote_signals/connector.nim

```diff
@@ -0,0 +1,28 @@
+import json, tables, chronicles
+import base
+
+include  app_service/common/json_utils
+
+type ConnectorSendRequestAccountsSignal* = ref object of Signal
+  dAppUrl*: string
+  dAppName*: string
+  dAppIconUrl*: string
+
+type ConnectorSendTransactionSignal* = ref object of Signal
+  dAppUrl*: string
+  chainID*: int
+  txArgsJson: string
+
+proc fromEvent*(T: type ConnectorSendRequestAccountsSignal, event: JsonNode): ConnectorSendRequestAccountsSignal =
+  result = ConnectorSendRequestAccountsSignal()
+  result.dAppUrl = event["event"]{"dAppUrl"}.getStr()
+  result.dAppName = event["event"]{"dAppName"}.getStr()
+  result.dAppIconUrl = event["event"]{"dAppIconUrl"}.getStr()
+  echo "--------> ConnectorSendRequestAccountsSignal: ", result.dAppUrl, " ", result.dAppName, " ", result.dAppIconUrl
```

Comment:
leftover? Could be cnnverted to info I guess

Reply:
Apologies, i forgot to draft this PR, it's not for merging